gradle.properties

```diff
@@ -78,3 +79,8 @@ lr.docker.environment.namespace=LRWSDE
 # The number of cluster nodes to start up in addition to the main Liferay
 # instance. Setting this to 0 means there is no clustering.
 lr.docker.environment.cluster.nodes=0
+
+# Optional
+#
+# The comma-delimited list of hostnames (domains) used to access Liferay when using a webserver.
+lr.docker.environment.web.server.hostnames=my.example.com
```

Comment:
Same comment as above, I don't think we want to actually make this the default value.